docs/docs/self-hosting/manage/tls_modes.mdx

```diff
@@ -2,15 +2,25 @@
 title: TLS Modes
 ---
 
-To allow ZITADEL to be run on any kind of infrastructure it allows to configure on how tho handle TLS connections.
-There are three mode of operation: `external`, `enabled`, `disabled`.
+To run ZITADEL on any kind of infrastructure, you can configure on how to handle TLS connections.
+There are three modes of operation: `disabled`, `external`, `enabled`.
 
 Generally this command is set as argument while starting ZITADEL. For example like this:
 
 ```bash
 zitadel start-from-init --masterkey "MasterkeyNeedsToHave32Characters" --tlsMode disabled
 ```
 
+## Disabled
+
+With the mode `disabled`, you instruct ZITADEL to await all connections with plain http without TLS.
+
+:::caution
+
+Be aware this is not a secure setup and should only be used for test systems!
+
+:::
+
```

Comment:
was this section moved to the top on pupose? resp. is the main problem people are facing?
because i noticed you used this order also for the examples

Reply:
IMO it's an easier start when the brain doesn't have to immediately mind about TLS